src/System.Private.CoreLib/shared/System/SpanHelpers.Byte.cs

```diff
@@ -1648,21 +1661,21 @@ private static unsafe IntPtr GetByteVector256SpanLength(IntPtr offset, int lengt
             => (IntPtr)((length - (int)(byte*)offset) & ~(Vector256<byte>.Count - 1));
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static unsafe IntPtr UnalignedByteCountVector(ref byte searchSpace)
+        private static unsafe IntPtr UnalignedCountVector(ref byte searchSpace)
```

Comment:
Change because is typed by param so doesn't need `Byte` in its name; motivated when adding same methods for `char` in https://github.com/dotnet/coreclr/pull/22505